Makefile

```diff
@@ -139,8 +148,10 @@ update-gotestsum:
 
 update-proto-plugins:
 	@printf $(COLOR) "Install/update proto plugins..."
-	@go install -modfile build/go.mod github.com/temporalio/gogo-protobuf/protoc-gen-gogoslick
+	@go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
 	@go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```

Comment:
Curious why we're installing latest as opposed to using the modfile.

Reply:
No reason. I'd be happy to change that if the team cares